configmerge/repo_cache.go

```diff
@@ -0,0 +1,19 @@
+package configmerge
+
+type repoCache struct {
+	cache map[string]string
+}
+
+func NewRepoCache() RepoCache {
+	return repoCache{
+		cache: map[string]string{},
+	}
+}
+
+func (c repoCache) GetRepo(ref ConfigReference) string {
+	return c.cache[ref.RepoKey()]
+}
+
+func (c repoCache) SetRepo(dir string, ref ConfigReference) {
+	c.cache[ref.RepoKey()] = dir
+}
```

Comment:
Does it worth having this struct just to store a map? It seems too simple and this could move into the config reader.

If this entity would manage the local directory, folder creation and cleanup as well then I would say it is definitely a keeper.  But in this state I think we can get rid of it.

Reply:
Agree, updated in https://github.com/bitrise-io/bitrise/pull/982/commits/8490578d2728e5edafa029108ce70d2914debd5c